python/ray/serve/api.py

```diff
@@ -539,7 +539,7 @@ def run(
 
     for deployment in deployments:
         # Overwrite route prefix
-        if route_prefix != "/" and deployment._route_prefix:
+        if route_prefix is not DEFAULT.VALUE and deployment._route_prefix is not None:
```

Comment:
why do we check `deployment._route_prefix is not None`? if route_prefix is not default value, should always do `deployment._route_prefix = route_prefix`?

Reply:
`deployment._route_prefix is not None` only for the ingress deployment.